src/Symfony/Component/OpenApi/Configurator/Traits/ExternalDocsTrait.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\OpenApi\Configurator\Traits;
+
+use Symfony\Component\OpenApi\Model\ExternalDocumentation;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ * @author Selency Team <tech@selency.fr>
+ */
+trait ExternalDocsTrait
+{
+    private ?ExternalDocumentation $externalDocs = null;
+
+    public function externalDocs(string $url, string $description = null, array $specificationExtensions = []): static
```

Comment:
I suggest adding the phpdoc to describe the type of the array (based on the ExtensionsTrait, I suppose it is `array<string, mixed>`)